app/graphql/types/availability_type.rb

```diff
@@ -5,7 +5,7 @@ class AvailabilityType < Types::BaseObject
     field :start_date_time, String, null: false
     field :end_date_time, String, null: false
     field :status, String, null: false
-    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
-    field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :created_at, String, null: false
+    field :updated_at, String, null: false
```

Comment:
When this was DateTime, the test wasn't passing because the dates were slightly off. We could look into this more but for now I'll just pass as a string.